packages/react-composites/tests/browser/chat/app/webpack.config.js

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+const path = require('path');
+const webpack = require('webpack');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+const Dotenv = require('dotenv-webpack');
+
+module.exports = {
+  entry: path.join(__dirname, 'index.tsx'),
+  devtool: 'eval-source-map',
+  resolve: {
+    extensions: ['.ts', '.tsx', '.js'],
+    alias: {
+      // reference internal packlets src directly for hot reloading when developing
+      'react-components': path.resolve(__dirname, '../../../../../react-components/src')
+    }
+  },
+  output: {
+    path: path.join(__dirname, 'dist'),
+    filename: 'build.js'
+  },
+  module: {
+    rules: [
+      {
+        test: /\.tsx?$/,
+        loader: 'ts-loader',
+        options: {
+          transpileOnly: true
+        },
+        exclude: /dist/
+      },
+      {
+        test: /\.css$/i,
+        use: ['style-loader', 'css-loader']
+      },
+      {
+        test: /\.svg/,
+        type: 'asset/inline'
+      }
+    ]
+  },
+  plugins: [
+    new HtmlWebpackPlugin({ template: path.join(__dirname, 'public', 'index.html') }),
+    new Dotenv({ path: path.join(__dirname, '.env') })
```

Comment:
I was pretty sure I saw this .env file somewhere, is it get removed?

One more question is there a way to override env value by running some command line? I guess this overriding ability will be needed when we are setting up our CI

Reply:
I need to figure that out somehow. In any case, we will be able to pipe an env variable to an .env file through CLI to make it work. But I will invest more time when we get to that stage.